### PR TITLE
feat: Implement account removal utility

### PR DESCRIPTION
**Pull request by haroon-schmrkow**
This PR adds ability of triggering a `disableModule` call in order to remove a signer/owner of the account.

### CODE REVIEW

File: src/entrypoints/accounts-actions/accountRemoval.ts [hunk 1]
```diff
@@ -0,0 +1,127 @@
...
+ *
+ * const dispatchTx = await populateRemoveOwnerDispatch(
+ *  { account: `0x<address>` },
+ *  newOwner: `0x${string}`,
+ * );
+ * await relayer.sendTransaction(enqueueTx);
+ */
+export function populateRemoveOwnerDispatch(
+  { account }: DispatchParameters,
+  removalConfig: RemovalConfig
```
Comment (cristovaoth) [inline src/entrypoints/accounts-actions/accountRemoval.ts:100]: call this config

File: src/entrypoints/accounts-actions/accountRemoval.ts [hunk 2]
```diff
@@ -0,0 +1,127 @@
...
+ *  newOwner: `0x${string}`,
+ *  // callback that wraps an eip-712 signature
+ *  ({ domain, primaryType, types, message }) =>
+ *    owner.signTypedData(domain, primaryType, types, message)
+ * );
+ * await relayer.sendTransaction(enqueueTx);
+ */
+export async function populateRemoveOwnerEnqueue(
+  { account, chainId, salt }: EnqueueParameters,
+  removalConfig: RemovalConfig,
```
Comment (cristovaoth) [inline src/entrypoints/accounts-actions/accountRemoval.ts:66]: call it config

File: src/entrypoints/accounts-actions/accountRemoval.ts [hunk 3]
```diff
@@ -0,0 +1,127 @@
...
+
+  return populateExecuteDispatch(
+    { account },
+    createInnerTransaction(account, removalConfig)
+  );
+}
+
+export function createInnerTransaction(
+  account: string,
+  { previousLastAccountOwner, accountToRemove }: RemovalConfig
```
Comment (cristovaoth) [inline src/entrypoints/accounts-actions/accountRemoval.ts:112]: I would call this prevOwner and owner

**Review by cristovaoth — CHANGES_REQUESTED** [review]
I would say you either cram these new populateEnqueue and populateDispatch into actions/accountOwner

Or you split them code in three files:
- actions/getAccountOnwers
- actions/addAccountOwner
- actions/removeAccountOwner

Those 3 files should export 5 functions

File: src/entrypoints/accounts-actions/accountRemoval.ts (hunk 3 above)
Comment (haroon-schmrkow) [reply src/entrypoints/accounts-actions/accountRemoval.ts:112]: `owner` is kind of unexplicit I'd say, `ownerToRemove` maybe?

Comment (cristovaoth) [reply src/entrypoints/accounts-actions/accountRemoval.ts:112]: or prev / owner. I guess additional explanations should come on the code block that precedes the functions. Keeping the var names lean is easier on the reading eye

**Pull request merged**